### ISSUE TITLE
Conference outcomes

### ISSUE THREAD

**Issue by dasaderi** [OpenCon-Cascadia/organization#3]
Intended conference outcomes as discussed at first meeting: 
* Engagement with and training for early career researchers, student and faculty around publishing, scholarly outputs, and using open methods throughout the research process.

* Create a multidisciplinary community.  Make sure that librarians, faculty, and individuals from different  backgrounds and expertise interact and partner.

* Balance between skill building and getting people excited about open scholarship/science.

* Collaboration in order to realize success for new and existing open oriented projects.

* Connect people with concrete outcomes and ongoing relationships

**Comment by dasaderi** [comment]
@lwinfree and @apreshill any progress with the website/splash page?

**Comment by lwinfree** [comment]
@dasaderi Yes :-) The website is here: https://opencon-cascadia.github.io/
and [here](https://github.com/OpenCon-Cascadia/OpenCon-Cascadia.github.io/issues) is the issue tracker for the website (please add issues for things you'd like changed)

**Comment by dasaderi** [comment]
@lwinfree and @apreshill, it's AMAZING! Thank youuuu! :dancing_women:

**Comment by dasaderi** [comment]
At our last call, Jill suggested to also map out "unintended consequences". e.g., we don't want the conference to only have librarians, or only have researchers. We need to identify our assumptions and ensure we address them best we can within our team.
